webui.py

```diff
@@ -88,44 +92,67 @@ def initialize():
     modules.scripts.load_scripts()
 
     modules.sd_models.load_model()
-    shared.opts.onchange("sd_model_checkpoint", wrap_queued_call(lambda: modules.sd_models.reload_model_weights(shared.sd_model)))
-    shared.opts.onchange("sd_hypernetwork", wrap_queued_call(lambda: modules.hypernetworks.hypernetwork.load_hypernetwork(shared.opts.sd_hypernetwork)))
-    shared.opts.onchange("sd_hypernetwork_strength", modules.hypernetworks.hypernetwork.apply_strength)
+    shared.opts.onchange(
+        "sd_model_checkpoint",
+        wrap_queued_call(
+            lambda: modules.sd_models.reload_model_weights(shared.sd_model)
+        ),
+    )
+    shared.opts.onchange(
+        "sd_hypernetwork",
+        wrap_queued_call(
+            lambda: modules.hypernetworks.hypernetwork.load_hypernetwork(
+                shared.opts.sd_hypernetwork
+            )
+        ),
+    )
+    shared.opts.onchange(
+        "sd_hypernetwork_strength", modules.hypernetworks.hypernetwork.apply_strength
+    )
 
     # make the program just exit at ctrl+c without waiting for anything
     def sigint_handler(sig, frame):
-        print(f'Interrupted with signal {sig} in {frame}')
+        print(f"Interrupted with signal {sig} in {frame}")
         os._exit(0)
 
     signal.signal(signal.SIGINT, sigint_handler)
 
 
 def create_api(app):
     from modules.api.api import Api
+
     api = Api(app, queue_lock)
     return api
 
+
 def wait_on_server(demo=None):
     while 1:
         time.sleep(0.5)
-        if demo and getattr(demo, 'do_restart', False):
+        if demo and getattr(demo, "do_restart", False):
             time.sleep(0.5)
             demo.close()
             time.sleep(0.5)
             break
 
+
 def api_only():
     initialize()
 
     app = FastAPI()
     app.add_middleware(GZipMiddleware, minimum_size=1000)
     api = create_api(app)
 
-    api.launch(server_name="0.0.0.0" if cmd_opts.listen else "127.0.0.1", port=cmd_opts.port if cmd_opts.port else 7861)
+    api.launch(
+        server_name="0.0.0.0" if cmd_opts.listen else "127.0.0.1",
+        port=cmd_opts.port if cmd_opts.port else 7861,
+    )
 
 
 def webui():
     launch_api = cmd_opts.api
+
+    download_repos()

```

Comment:
The rest of the changes are formatting updates, but this is where I call the methods that I introduced as part of this PR.